### PR TITLE
[Feature] Support to Item options name

### PR DESCRIPTION
**Pull request by alisonrag**
Support to Options name.
Update in command item dec

TODO:
- [ ]  Code review
- [x] Fix bug in kore after move a item from storage/cart to inventory, kore remove the options name

Before:
image

After:
image

image

### CODE REVIEW

**Comment by Nipodemos** [comment]
conflicts resolved

File: src/Globals.pm [hunk 1]
```diff
@@ -26,7 +26,8 @@ use Modules 'register';
 # Do not use any other Kore modules here. It will create circular dependancies.
 
 our %EXPORT_TAGS = (
-	config  => [qw(%arrowcraft_items %avoid @chatResponses %cities_lut %config %consoleColors %directions_lut %equipTypes_lut %equipSlot_rlut %equipSlot_lut %haircolors @headgears_lut @msgTable %items_control %items_lut %itemSlotCount_lut %itemsDesc_lut %itemTypes_lut %jobs_lut %maps_lut %masterServers %monsters_lut %npcs_lut %packetDescriptions %portals_lut @portals_lut_missed %responses %sex_lut %shop %buyer_shop %skillsDesc_lut %lookHandle %skillsArea %skillsEncore %spells_lut %emotions_lut %timeout $char %mon_control %priority %routeWeights %pickupitems %rpackets %itemSlots_lut %statusHandle %statusName %effectName %hatEffectHandle %hatEffectName %portals_los %stateHandle %ailmentHandle %mapTypeHandle %mapPropertyTypeHandle %mapPropertyInfoHandle %elements_lut %mapAlias_lut %quests_lut $Blacksmith_Blessing %itemStackLimit %title_lut)],
+
```
Comment (sctnightcore) [inline src/Globals.pm]: del it ?

Comment (Nipodemos) [reply src/Globals.pm]: If I understand correctly, he just added 1 blank line

**Comment by alisonrag** [comment]
image

**Comment by hubertyang88** [comment]
How to switch weapon when using item options ? weapon [fire] and weapon [water] looks same

**Comment by royalgarter** [comment]
> How to switch weapon when using item options ? weapon [fire] and weapon [water] looks same

Can I pump this. Currently the Crimson items is popular for leveling. I have to worked arround by using multi upgraded level for distinct the items for switching using "autoSwitch"

**Pull request merged**